Source/Core/VideoCommon/PixelShaderGen.h

```diff
@@ -62,6 +64,7 @@ struct pixel_shader_uid_data
 	u32 forced_early_z : 1;
 	u32 early_ztest : 1;
 	u32 bounding_box : 1;
+	u32 zfreeze : 1;
```

Comment:
So, now this block is 33 bits big. Please try to not change the bit alignment in this struct.